etc/ui-config/mh_default_org/paella7/config.json

```diff
@@ -228,6 +222,18 @@
             ],
             "tabIndexStart": 40
         },
+        "org.opencast.paella.multiVideoDynamicLayout": {
+            "enabled": true,
+            "validContent": [
+                {
+                    "id": "multi-video",
+                    "content": [],
+                    "icon": "present-mode-4.svg",
+                    "title": "Auto"
+                }
+            ],
+            "tabIndexStart": 40
```

Comment:
I don't really understand these configuration options.
Icon doesn't seem to be used. Is it necessary?

The title seems to be what's being shown in the video layout selector. That may make sense to configure:
<img width="250px" src="https://github.com/opencast/opencast/assets/1008395/68d3eaa2-32ca-4aa3-abea-dc44c6eabecf" />


But everything else could be just hard-coded configuration. Or does it make sense to configure this array differently at some point?

So, you could just have something like this as configuration which is less confusing for users:
```json
{
  "org.opencast.paella.multiVideoDynamicLayout": {
    "enabled": true,
    "title": "Auto",
    "icon": "present-mode-4.svg"  ← not sure about this?
  },
  "tabIndexStart": 40
}
```

Reply:
I'm on vacation right now, I'll review this PR when I get back.

> Icon doesn't seem to be used. Is it necessary?
> 
> The title seems to be what's being shown in the video layout selector. That may make sense to configure:

Just to clarify, icons are not used in default settings, but this can be changed in the `config.json` file
https://github.com/opencast/opencast/blob/f4454f61f2f66c9166384fee13b27d0e86df42cb/etc/ui-config/mh_default_org/paella7/config.json#L347-L357

Reply:
I kept `content` because I am not sure if this could break something. In https://github.com/polimediaupv/paella-core/blob/ea62b0b9a5a99d8344069d20eb42d03e143d35d5/src/js/core/VideoLayout.js#L38C80-L38C80 this config is used. Without this configuration the player with the default settings seems to work.